### PR TITLE
Update malformed URL error msg for options page

### LINKED ISSUE #39: Provide details on a valid URL for the options page form
When an invalid string is provided inside of the options form text, the following error is displayed:

image

It would be great if the options page somewhere indicated what the correct format for the URL should be.

Possible solution could be to provide a short description and example in the error message.

Rough mockup:
image

### PR DESCRIPTION
**Pull request by dkwon17**
Fixes #39 

The new error message:
image

To test this PR,
1. Checkout this PR and run `yarn:build`
2. Sideload the built extension located in `dist/chromium` into Google Chrome
3. Open the options page, and enter invalid content into the textbox 
4. Run `yarn test` to run all tests`

### CODE REVIEW

File: src/options/App.tsx [hunk 1]
```diff
@@ -111,6 +115,23 @@ export const App = () => {
...
+            <SplitItem>
+                <ExclamationCircleIcon
+                    color="var(--pf-global--danger-color--100)"
+                    className="pf-u-mr-xs"
+                />
+            </SplitItem>
+            <SplitItem>
+                <div className="pf-c-form__helper-text pf-m-error">
+                    Provide the URL of your Dev Spaces installation, e.g.,
+                    https://devspaces.mycluster.redhat.com
```
Comment (nickboldt) [inline src/options/App.tsx:129]: I wonder if this should be fully generic, without an actual company here? 
```suggestion
                    https://devspaces.mycluster.mycorp.com
```

File: src/options/__tests__/App.spec.tsx [hunk 2]
```diff
@@ -117,3 +117,14 @@ it("should disable 'Add' button when invalid input is entered in the input box",
...
+
+it("should display error message when invalid input is entered in the input box", async () => {
+    preferencesMock.setEndpoints([
+        { url: "https://url-1.com", active: true, readonly: true },
+    ]);
+
+    const { findByPlaceholderText, findByText } = render(<App />);
+    const inputBox = await findByPlaceholderText("Add endpoint");
+    fireEvent.change(inputBox, {target: {value: "invalidtext"}})
+    await findByText("Provide the URL of your Dev Spaces installation, e.g., https://devspaces.mycluster.redhat.com");
```
Comment (nickboldt) [inline src/options/__tests__/App.spec.tsx:129]:
```suggestion
    await findByText("Provide the URL of your Dev Spaces installation, e.g., https://devspaces.mycluster.mycorp.com");
```

**Review by ibuziuk — APPROVED** [review]

**Comment by dkwon17** [comment]
Thank you @nickboldt @ibuziuk, I will merge shortly

**Comment by dkwon17** [comment]
Here is the new text:
image

**Review by nickboldt — APPROVED** [review]

**Pull request merged**